src/cljs/rems/actions/add_licenses.cljs

```diff
@@ -84,16 +82,12 @@
                      :on-comment on-set-comment}]
     [:div.form-group
      [:label (text :t.actions/licenses-selection)]
```

Comment:
All the labels need a `for` attribute so that clicking the label fill focus the field. This wasn't possible with the old component, but the new one has a proper input field which can be focused.

Reply:
Done.